input/fsh/omrs-person.fsh

```diff
@@ -0,0 +1,16 @@
+Profile: OMRSPerson
+Parent: Person
+Id: omrs-person
+Title: "OpenMRS Person"
+Description: "A FHIR Person as understood by OpenMRS"
+* identifier 0..*
+* name 1..* MS
+* telecom 0..* MS
+* gender 1..1 MS
+* birthDate 1..1 MS
+* address 0..* MS
+* active 1..1 MS
+* active = true
```

Comment:
is this currently always the case? Every patient returned by the FHIR service is active?

Reply:
Yes. OMRS doesn’t really distinguish between “active” or “inactive” other than being “voided” or “non-voided” and we treat voided data as if it’s been deleted. I think this attribute is only implemented because initially `active = !voided` and, IIRC, `active` is used as a search parameter to find voided patient records.